contracts/VoucherSets.sol

```diff
@@ -10,7 +10,7 @@ import "./interfaces/IVoucherKernel.sol";
 import "./interfaces/IVoucherSets.sol";
 import "./interfaces/ICashier.sol";
 
-//preparing for ERC-1066, ERC-1444, EIP-838
+//preparing for ERC-1066, ERC-1444
```

Comment:
Whole this line can be removed, since both ERC-1066, ERC-1444 are Stagnant and we are not following that at all.

Reply:
removed